backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoice/request/CreateManualInvoiceRequest.java

```diff
@@ -78,6 +79,8 @@ public class CreateManualInvoiceRequest
 	@NonNull
 	CurrencyId currencyId;
 
+	@NonNull PaymentTermId paymentTermId;
```

Comment:
i think this shall be nullable... see ur code changes from ManualInvoiceService

Reply:
C_Invoice.C_PaymentTerm_ID is mandatory. Changed logic in ManualInvoiceService.